client/src/components/Authenticate.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Authenticate extends Component
+{
+    static propTypes = {
```

Comment:
Our version of babel doesn't support this syntax:

https://babeljs.io/docs/en/babel-plugin-proposal-class-properties

You can `yarn add --dev` that but then you'll run into requiring Babel 7.0. We use 6.26. Currently this is defined with our dependency to `@silverstripe/webpack-config`. I would not recommend going down the rabbit hole of upgrading that. I'm not sure how incompatible things will get if you try to remove our dependency on `@silverstripe/webpack-config`...